src/app/services/push.service.ts

```diff
@@ -50,25 +49,29 @@ export class PushService {
   }
 
   public register() {
-    new PushRegistration(new ConfigurationService(config))
-    .register({
-      alias: 'cordova',
-      categories: ['ionic', 'showcase']
-    }).then(() => {
-      console.log('Push registration successful');
-    }).catch((err) => {
-      console.warn('Push registration unsuccessful');
-    });
+    if (!this.configService.getPushConfig()) {
```

Comment:
Should this condition be happening here? Now this function has two responsibilities.

Reply:
It it was disabled, but I have made fixes in push now so it will work